### DISCUSSION TITLE
Apsolute Beginer in GitHub and C#

### DISCUSSION THREAD

**Discussion by 2311627BorisBacic** [melanchall/drywetmidi#302] [General]
I have install Visual Studio on Windows 10 laptop then I go to Github download drywetmidi in which dir I must put unziped data to works with C# Visual Studio. Then can I simply start with copy and paste code from draywet examples to C# in Visual Studio 
Thanks lot any sugestions

**Comment by 2311627BorisBacic** [comment]
I want to get chords changes in MIDI file with time of change in miliseconds from start in specific format [00:00.00] (chord name like G).
Time format is in first two 00 minutes,second seconds from 00 to 60, third 00 miliseconds from 0 to 100

**Comment by melanchall** [comment]
Hi,

GitHub projects is not the place where you will be taught how to code and use technologies for your programming languages. I can help with my library only. And please show your attempts to solve the task. And then I'll point you to issues in your code.

> I must put unziped data to works with C# Visual Studio

No. You need to use NuGet, read about it.

Please read the library [documentation](https://melanchall.github.io/drywetmidi/).

**Reply by 2311627BorisBacic** [reply]
Thanks lot a read and come back

**Comment by 2311627BorisBacic** [comment]
using Melanchall.DryWetMidi.Core;
using Melanchall.DryWetMidi.Interaction;

// Read a MIDI file
var midiFile = MidiFile.Read("Cool song.mid");

// Manage timed events of the first track chunk of the file
using (var objectsManager = midiFile.Chunks
                                    .OfType<TrackChunk>()
                                    .First()
                                    .ManageTimedEvents())
{
    // Get timed events ordered by time
    var events = objectsManager.Objects;
    
    // Set absolute time of the first Lyric event
    var firstLyricEvent = events.FirstOrDefault(e => e.Event is LyricEvent);
    if (firstLyricEvent != null)
        firstLyricEvent.Time = 2000;

    // Add new Pitch Bend event with absolute time = 3000
    events.Add(new TimedEvent(new PitchBendEvent(8000), 3000));
}


// After this I tried simple read of midi file

var midiFile = MidiFile.Read("Some great song.mid");

 After this in Visual Studio I get one 1 error

Two output file names resolved to the same output path: "obj\x86\Debug\Proba8.Resources.resources"	Proba8


I read something about overflow of stack how this can be solved

**Reply by melanchall** [reply]
First of all, read this article - [Support](https://melanchall.github.io/drywetmidi/articles/dev/Support.html). You'll find there info about how to create good issues so I can (and have willing) to help you.

And second,

> After this in Visual Studio I get one 1 error
> Two output file names resolved to the same output path: "obj\x86\Debug\Proba8.Resources.resources" Proba8

How is **your** resources file related to DryWetMIDI? The answer is: it's not related at all. I can help you with my library only. Any other problems with your project should be handled by you. Or you can ask a question on Stack Overflow or so.
